core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -568,7 +568,7 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging with Seria
     // If spark.executor.heartbeatInterval bigger than spark.network.timeout,
     // it will almost always cause ExecutorLostFailure. See SPARK-22754.
     require(executorTimeoutThresholdMs > executorHeartbeatIntervalMs, "The value of " +
-      s"${networkTimeout}=${executorTimeoutThresholdMs}ms must be no less than the value of " +
+      s"${networkTimeout}=${executorTimeoutThresholdMs}ms must be greater than the value of " +
```

Comment:
Isn't it virtually same?

Reply:
"no less" than is slightly confusing as the application fails if both values are same, hence changing it to greater. Please let me know if it makes sense.

Reply:
Okay. it's probably because English isn't my first language. cc @srowen.

Reply:
"no less" would be "greater than or equal to" and the check does look for "greater than" so the correction is OK.

Reply:
@srowen Thanks for checking it out, yes, that was the intent. Could you please help in closing this PR?